applications/ParticleMechanicsApplication/tests/beam_tests/hyperelastic_cantilever_beam/penalty_self_weight_load_2D_quad_test_parameters.json

```diff
@@ -56,17 +56,18 @@
             "Parameters"    : {
                 "model_part_name" : "Background_Grid.PENALTY_Condition_Auto1",
                 "particles_per_condition" : 7,
-                "penalty_factor"  : 1e15,
+                "penalty_factor"  : 1e20,
```

Comment:
why is that?

Reply:
sorry I can revert that it also works for 1e15